tests/projects/test_utils.py

```diff
@@ -105,6 +106,21 @@ def test__fetch_git_repo(local_git_repo, local_git_repo_uri, version, expected_v
     assert repo.active_branch.name == expected_version
 
 
+@pytest.mark.parametrize(
+    "commit",
+    # Fetch the most recent two commits
+    requests.get("https://api.github.com/repos/mlflow/mlflow-example/commits").json()[:2],
+)
+def test_fetch_git_repo_commit(tmp_path, commit):
+    _fetch_git_repo(
+        "https://github.com/mlflow/mlflow-example.git",
+        commit["sha"],
```

Comment:
Is this a List[Dict]? Which element is accessing the key 'sha'?

Reply:
ahhh nevermind. It will generate two parameterized values to validate pulling an explicit non-head commit. Great stuff!